requirements.txt

```diff
@@ -1,3 +1,3 @@
 jsonschema >= 4.17
```

Comment:
Consider pin jsonschema version since 4.18.0 version might require Rust compiler :|

Reply:
Hi Amanda, as we decided to use the ingress relation instead of nginx-route. I will close this pull request and pin the jsonschema version in the next pull request. Thanks!